### ISSUE TITLE
Umbraco Api throws internall null point on       getByIds method

### ISSUE THREAD

**Issue by bielu** [umbraco/Umbraco-CMS#12273]
### Which *exact* Umbraco version are you using? For example: 9.0.1 - don't just write v9

9.4.3

### Bug summary

We created in v8 editor which use contentResource and  we used `contentResource.getByIds(ids)`
so our code looks like that:
``` 
    contentResource.getByIds(ids)
                    .then(function (entities) {
                        var teasers = mapTeasers(entities).slice(0, $scope.control.value.count);
                        return mapImages(teasers);
                    });
```
but seems to have issues with nucache. 

### Steps to reproduce

1) create editor which use    contentResource.getByIds(ids)
2) try add editor
3) see modal error with no error specifics

### Expected result / actual result

Editor actually works and has no modal errors

**Comment by bielu** [comment]
@nikolajlauridsen it was fixed in 9.5 (by incident but was 😂)

**Comment by nikolajlauridsen** [comment]
Ahh lovely, I'm glad to hear that 🎉, Of course, unfortunate that we didn't get to it before, sorry about that. 

But I'll go ahead and close this one this :D Thanks for reporting
